model_analyzer/device/gpu_device_factory.py

```diff
@@ -18,7 +18,10 @@
 import model_analyzer.monitor.dcgm.dcgm_structs as structs
 from model_analyzer.model_analyzer_exceptions import TritonModelAnalyzerException
 
+from pynvml import *
+
 import numba.cuda
```

Comment:
can this be removed?

Reply:
No, numba.cuda is still being used

Reply:
I'm confused. I thought the whole point of this was to remove DCGM, but it doesn't appear that dcgm was used in the function you removed? I do still see it used in init_all_devices() though. Did we change the wrong code?